doc/nep1.md

```diff
@@ -256,34 +256,50 @@ Coding Conventions
 Conventions for multi-line statements and expressions
 -----------------------------------------------------
 
-- Tuples which are longer than one line should indent their parameters to
-  align with the parameters above it.
+- Tuples which are longer than one line should split their parameters across multiple lines, so that the definition resembles a standard "object" type definition.
 
     ```nim
     type
-      LongTupleA = tuple[wordyTupleMemberOne: int, wordyTupleMemberTwo: string,
-                         wordyTupleMemberThree: float]
+      LongTupleA = tuple[
+        wordyTupleMemberOne: int,
+        wordyTupleMemberTwo: string,
+        wordyTupleMemberThree: float
+      ]
     ```
 
 - Similarly, any procedure and procedure type declarations that are longer
   than one line should do the same thing.
 
     ```nim
     type
-      EventCallback = proc (timeReceived: Time, errorCode: int, event: Event,
-                            output: var string)
-
-    proc lotsOfArguments(argOne: string, argTwo: int, argThree: float,
-                         argFour: proc(), argFive: bool): int
-                        {.heyLookALongPragma.} =
+      EventCallback = proc (
+        timeReceived: Time,
+        errorCode: int,
+        event: Event,
+        output: var string
+      )
+
+    proc lotsOfArguments(
+      argOne: string,
+      argTwo: int,
+      argThree: float,
+      argFour: proc(),
+      argFive: bool
+    ): int 
+      {.heyLookALongPragma.} =
```

Comment:
That is not how we do it. The NEP should reflect reality and not your personal preferences.